packages/atlas/src/providers/joystream/provider.tsx

```diff
@@ -87,7 +99,7 @@ export const JoystreamProvider: React.FC = ({ children }) => {
   }, [activeAccountId, accounts, initialized])
 
   return (
-    <JoystreamContext.Provider value={{ joystream: initialized ? joystream.current : undefined, proxyCallback }}>
+    <JoystreamContext.Provider value={{ joystream: initialized ? joystream.current : undefined, proxyCallback, price }}>
```

Comment:
This exposes the price but I would expect some hook to convert between tJOY's and formatted dollar string. Please take a look at what Bartosz did here: https://github.com/Joystream/atlas/pull/2097 So some hook for conversion would be useful.

Reply:
I added `useTokenPrice` below